services/cd-service/pkg/repository/db.go

```diff
@@ -0,0 +1,130 @@
+/*This file is part of kuberpult.
+
+Kuberpult is free software: you can redistribute it and/or modify
+it under the terms of the Expat(MIT) License as published by
+the Free Software Foundation.
+
+Kuberpult is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+MIT License for more details.
+
+You should have received a copy of the MIT License
+along with kuberpult. If not, see <https://directory.fsf.org/wiki/License:Expat>.
+
+Copyright 2023 freiheit.com*/
+
+package repository
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+	"github.com/freiheit-com/kuberpult/pkg/logger"
+	"github.com/golang-migrate/migrate/v4"
+	sqlite "github.com/golang-migrate/migrate/v4/database/sqlite3"
+	"github.com/golang-migrate/migrate/v4/source/file"
+	"github.com/golang-migrate/migrate/v4/source/iofs"
+	"go.uber.org/zap"
+	"math/rand"
+	"path"
+	"time"
+)
+
+func GetDBConnection(dbFolderLocation string) (*sql.DB, error) {
+	return sql.Open("sqlite3", path.Join(dbFolderLocation, "db.sqlite")) //not clear on what is needed for the user and password
```

Comment:
```suggestion
	return sql.Open("sqlite3", path.Join(dbFolderLocation, "db.sqlite"))
```

We can ignore the db authentication as long as we're just working with sqlite